tests/bsim/bluetooth/host/adv/resume2/connectable/_build.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+set -eu
+dotslash="$(realpath "$(dirname "${BASH_SOURCE[0]}")")"
+bin_dir="${BSIM_OUT_PATH}/bin"
+BOARD="${BOARD:-nrf52_bsim}"
```

Comment:
This will rot really fast (between other things there is a plan to deprecate the usage of "short" board identifiers by the build maintainer)
If you want to keep in tree your own mini build scripts, better copy paste this:

https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/bsim/bluetooth/host/privacy/central/test_scripts/_compile.sh

The same applies to the other build scripts.